app/src/main/res/values/strings.xml

```diff
@@ -367,4 +367,9 @@
     <string name="helpdesk_email_error">Please enter a valid email</string>
     <string name="dialog_discard_positive">Discard</string>
     <string name="dialog_discard_inputs">Do you want to discard of your input?</string>
+
+    <string name="item_duration_hour_minute">%1$d hr, %2$d min</string>
```

Comment:
Should there be a comma in the English locale? Because in the German, there isn't.

Reply:
True, but why did it work? 🤔

Reply:
It's fine this way. We discussed that in person together with the iOS examples. In German we use abbreviations, in English it's more like a unit.